src/controllers/handlers/http/update-community-partially-handler.ts

```diff
@@ -0,0 +1,62 @@
+import { InvalidRequestError, NotAuthorizedError } from '@dcl/platform-server-commons'
+import { HandlerContextWithPath, HTTPResponse } from '../../../types'
+import { errorMessageOrDefault } from '../../../utils/errors'
+import { CommunityNotFoundError } from '../../../logic/community/errors'
+
+type UpdateCommunityPartiallyBody = {
+  editorsChoice?: boolean
+}
+
+export async function updateCommunityPartiallyHandler(
+  context: Pick<
+    HandlerContextWithPath<'communities' | 'logs', '/v1/communities/:id'>,
+    'components' | 'params' | 'verification' | 'request'
+  >
+): Promise<HTTPResponse> {
+  const {
+    components: { communities, logs },
+    params: { id: communityId },
+    verification,
+    request
+  } = context
+
+  const logger = logs.getLogger('update-community-partially-handler')
+  const userAddress = verification!.auth.toLowerCase()
+
+  try {
+    const body: UpdateCommunityPartiallyBody = await request.json()
```

Comment:
May we can use this shared component to handle these validations:
https://github.com/decentraland/core-components/blob/main/components/schema-validator/src/component.ts